sport/app/cricket/feed/CricketThrottler.scala

```diff
@@ -0,0 +1,43 @@
+package cricket.feed
+
+import java.util.concurrent.TimeUnit
+
+import akka.NotUsed
+import akka.actor.{Actor, ActorRef, ActorSystem, Props}
+import akka.pattern.{ask, pipe}
+import akka.stream.scaladsl.{Sink, Source}
+import akka.stream.{ActorMaterializer, OverflowStrategy, ThrottleMode}
+import akka.util.Timeout
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.duration._
+import scala.reflect.ClassTag
+
+case class CricketThrottledTask[+T](task: () => Future[T])
+
+class CricketThrottlerActor()(implicit materializer: ActorMaterializer) extends Actor {
+  import context.dispatcher
+
+  private case class TaskWithSender[+T](sender: ActorRef, task: () => Future[T])
+
+  val throttler: ActorRef = Source.actorRef[CricketThrottledTask[Nothing]](bufferSize = 1024, OverflowStrategy.dropNew)
```

Comment:
Can you give more details about why you chose those values for `bufferSize` and `OverflowStrategy`